apis/sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/BuiltInFunctionResource.java

```diff
@@ -113,7 +113,7 @@ public class BuiltInFunctionResource {
       })
   @POST
   @Path(
-      "{collection:\\w+}/{document-id}{slash: /?}{document-path: (.*)?}/function{trailingSlash: /?}")
+      "{collection:\\w+}/{document-id: [^/]+}{slash: /?}{document-path: (.+)?}/function{trailingSlash: /?}")
```

Comment:
On `document-id`: would slash actually be allowed otherwise? I assumed it usually wouldn't. But I assume that it would be (otherwise wouldn't have need to prevent it)

Reply:
Yea I noticed that because of how quarkus URI parsing works with slashes, it actually slurps everything between literal `/` for each token. So I was getting matches on `{document-id: .+}{slash: /?}{document-path: (.+)?}` that were actually matching `abcde/fghij` as the value of document-id (which is ...not right)

Reply:
Yeah, just odd how it would include slash within if you do NOT include pattern. I would expect it to essentially use "anything but slash". But I guess not :)